### PR TITLE
added single order search and documentation

### PR DESCRIPTION
**Pull request by vaiibm**
please review and merge my code to main branch

### CODE REVIEW

File: src/main/java/com/ibm/searchorder/OrderSearchController.java [hunk 1]
```diff
@@ -13,11 +13,23 @@
 	@Autowired
 	OrderService orderService;
 
+	/**
+	 * Method to get order details with given id
+	 * 
+	 * @param orderID
+	 * @return
+	 */
+
```
Comment (hitler420) [inline src/main/java/com/ibm/searchorder/OrderSearchController.java:22]: /done by Arun

Comment (vaiibm) [reply src/main/java/com/ibm/searchorder/OrderSearchController.java:22]: ok

Comment (vaiibm) [reply src/main/java/com/ibm/searchorder/OrderSearchController.java:22]: now  resolve my code

**Review by hitler420 — APPROVED** [review]

**Comment by vaiibm** [comment]
done

**Pull request merged**